SSRadioButtonController/SSRadioButtonsController.swift

```diff
@@ -22,7 +22,7 @@ import UIKit
 class SSRadioButtonsController : NSObject
 {
     fileprivate var buttonsArray = [UIButton]()
-    fileprivate weak var currentSelectedButton:UIButton? = nil
+//    fileprivate weak var currentSelectedButton:UIButton? = nil
```

Comment:
ops... this needs line needs to be deleted...

Let me know if I update the pull request with a new commit or if you prefer to remove it during the merge.